spec/unit/facter/network_nexthop_ip_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Facter.fact(:network_nexthop_ip) do
```

Comment:
I'm not sure if this is the right way. In https://github.com/puppetlabs/pdk-templates/blob/main/object_templates/fact_spec.erb they favor a slightly different approach which looks cleaner to me.